hawc/apps/mgmt/urls.py

```diff
@@ -1,14 +1,13 @@
-from django.urls import include, path
+from django.urls import path
 from rest_framework.routers import SimpleRouter
 
-from . import api, views
+from . import views
 
 router = SimpleRouter()
```

Comment:
i think we can completely remove the router and delete the lines with it, nor more API!